locksmith/__tests__/controllers/userController.test.ts

```diff
@@ -101,7 +103,29 @@ describe('User Controller', () => {
     describe('when the provided email does not within the existing persistence layer', () => {
       it('returns an error code', async () => {
         let emailAddress = 'non-existing@example.com'
-        let response = await request(app).get(`/users/${emailAddress}/privatekey`)
+        let response = await request(app).get(
+          `/users/${emailAddress}/privatekey`
+        )
+        expect(response.statusCode).toBe(400)
+      })
+    })
+  })
+
+  describe("retrieving a user's recovery phrase", () => {
+    describe('when the user exists', () => {
+      it("returns the user's recovery phrase", async () => {
+        let response = await request(app).get(
+          '/users/user@example.com/recovery_phrase'
```

Comment:
since we use `privatekey`, can we use `recoveryphrase`?

Reply:
I know it is probably sub-optimal, but I'd rather err on the side of consistency.